internal/ent/enums/documenttype.go

```diff
@@ -13,12 +13,14 @@ var (
 	RootTemplate DocumentType = "ROOTTEMPLATE"
 	// Document are templates from root templates, or scratch, owned by the organization
 	Document DocumentType = "DOCUMENT"
+	// DocumentTypeInvalid is the default value for the DocumentType enum
+	DocumentTypeInvalid DocumentType = "INVALID"
 )
 
 // Values returns a slice of strings that represents all the possible values of the DocumentType enum.
 // Possible default values are "ROOTTEMPLATE", "DOCUMENT"
 func (DocumentType) Values() (kinds []string) {
-	for _, s := range []DocumentType{RootTemplate, Document} {
+	for _, s := range []DocumentType{RootTemplate, Document, DocumentTypeInvalid} {
```

Comment:
Same here, these are the valid values in the database, invalid should not be included 

```suggestion
	for _, s := range []DocumentType{RootTemplate, Document} {
```